gammapy/cube/sherpa_.py

```diff
@@ -1,13 +1,68 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import absolute_import, division, print_function, unicode_literals
 import numpy as np
+from ..utils.energy import EnergyBounds
+import sherpa.astro.ui as sau
+from sherpa.astro.ui import erf
+import astropy.wcs as pywcs
```

Comment:
Can you just explicitly import `WCS` here? I've never seen that convention of using `pywcs`